PowerEditor/src/ScintillaComponent/Buffer.h

```diff
@@ -300,15 +300,15 @@ class Buffer final {
 	void startMonitoring() { 
 		_isMonitoringOn = true; 
 		_eventHandle = ::CreateEvent(nullptr, TRUE, FALSE, nullptr);
-	};
+	}
 
 	HANDLE getMonitoringEvent() const { return _eventHandle; };
 
 	void stopMonitoring() { 
 		_isMonitoringOn = false;
 		::SetEvent(_eventHandle);
 		::CloseHandle(_eventHandle);
-	};

```

Comment:
I know it's not necessary to add `;`.
But it's just one of Notepad++ coding convention in *.h